### ISSUE TITLE
[In the News page] text edits

### ISSUE THREAD

**Issue by lesleykg1** [theBoardlist/website#28]
Couple of small edits
![in_the_news_-_theboardlist 2](https://cloud.githubusercontent.com/assets/16053877/11981780/a2fd9d22-a95a-11e5-8a66-628d0eb32d68.png)
![in_the_news_-_theboardlist](https://cloud.githubusercontent.com/assets/16053877/11981781/a2fe36f6-a95a-11e5-8caa-41bc232dad71.png)

**Comment by jimmybyrum** [comment]
@lesleykg1 I've made some updates, and also have a couple questions. Some of these are article titles, and some are quotes from the article. @eringallup and I were thinking the titles should be proper title case, and only the ones that are quotes should have quotes around them. And maybe all of them being consistent would be even better?

The 2nd Fortune article is neither the title of the article **or** a quote from the article. I changed it to be the title of the article.

Lastly, these articles seem to be in a random order- maybe it's the ones that they like more are higher up? Should these be in some order?

Anyway, if you still just want quotes around all of them, just say so. And let me know if we made the right call with that 2nd Fortune article.
